README.md

```diff
@@ -32,6 +26,14 @@ Install dependencies via `npm`:
 npm install
 ```
 
+## In this sample you will do the following:
+
+* Create a storage account.
+* Create a container.
+* Upload a stream to [blockblob](https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs).
+
+If you don't have a Microsoft Azure subscription, you can get a free trial account <a href="http://go.microsoft.com/fwlink/?LinkId=330212">here</a>.
```

Comment:
This trial account text should be in the same location as all the other samples.